src/main/api/2/studio-api-2.yaml

```diff
@@ -3315,6 +3315,11 @@ paths:
           schema:
             type: integer
             format: int64
+        - name: showIncompatible
+          in: query
+          description: Include incompatible plugins (default to false)
```

Comment:
defaults* to false

Is there no way of stating it's optionality and default value through actual definition arguments?

Reply:
yes, by default all parameters are optional (`required: false` is not needed), and it is possible to use `default: false`